packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts

```diff
@@ -74,6 +75,10 @@ export class SfdxCoreSettings {
     return this.getConfigValue(BETA_DEPLOY_RETRIEVE, false);
   }
 
+  public getLwcPreviewOnMobileEnabled(): boolean {
```

Comment:
Same as with the settings, this should be part of the lwc extension and not the core one.